### ISSUE TITLE
CLI tool to explain violation codes

### ISSUE THREAD

**Issue by Tapeline** [wemake-services/wemake-python-styleguide#3241]
# Feature request

## Reasoning

At the moment, if you want to learn more about a violation, the only option you have is go to the WPS website and read the corresponding section. However this is not really convenient to use and requires an internet connection (which may be unavailable in certain cases)

## Suggestion

Add a command like `wps explain <VIOLATION_CODE>` which will display the full violation description in terminal. 
```bash
$ wps explain WPS117
Forbid naming variables ``self``, ``cls``, or ``mcs``.

Reasoning:
|   These names are special, they should only be used as first
|   arguments inside methods.

Example:
|   # Correct:
|   class Test:
|       def __init__(self):
|           ...
|
|   # Wrong:
|   cls = 5
|   lambda self: self + 12

This rule checks: functions and methods.
Having any reserved names in ``lambda`` functions is not allowed.
```

Considering that each description is already provided in corresponding classes as a docstring, it should be fairly easy to implement.

## Existing solutions and takes on this problem

This was discussed in #1145, and the suggestion was to use [flakehell](https://github.com/life4/flakehell). It is now unmaintained. Its successor [flakeheaven](https://github.com/flakeheaven/flakeheaven) didn't update for 1.5 yrs (it's still on flake8 < v5.0.0, cite from pip: `flakeheaven 3.3.0 requires flake8<5.0.0,>=4.0.1, but you have flake8 7.1.1 which is incompatible`).

Another solution - [flake8-codes](https://github.com/orsinium-labs/flake8-codes/) - is even more out-of-date.

Want to hear your opinions on this: is it that of a need, should it be implemented in WPS? If we come to an agreement, I could implement it.
Thanks

**Comment by sobolevn** [comment]
Great idea, let's call this tool `wps` and subcommand can be `wps explain WPS111`

PRs are welcome!

**Comment by sobolevn** [comment]
Even better: we can add an option to automatically show all explanation in our formatter.

**Comment by Tapeline** [comment]
How should it be shown then? The explanations are quite big, so it may create a mess in console. If we want to implement this, maybe the only option is to display the explanations once for every new violation

**Comment by sobolevn** [comment]
Good question: I don't know yet :)

**Comment by Tapeline** [comment]
Maybe we output docstring as is? I think that parsing and formatting documentation is not really a thing we should, and, moreover, docstrings are already written in a pretty human-readable way

**Comment by sobolevn** [comment]
This way I would prefer to have both options: `--show-explanation=default: false` and `--show-explanation-command=default: false` (with `wps exaplain WPS` command)

This way we can show either the full text or just `Run: wps explain WPS_CODE to view the explanation`.

**Comment by sobolevn** [comment]
Let's start with `wps explain`, since it will show us how readable the whole output will be.

**Comment by Tapeline** [comment]
Okay 👍
